spec.md

```diff
@@ -1448,6 +1451,51 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 |-----------|-----------|-------------|-------------------|
 | Invalid `starting_token` | 10 ABORTED | Indicates that `starting_token` is not valid. | Caller SHOULD start the `ListVolumes` operation again with an empty `starting_token`. |
 
+#### `GetVolume`
+
+A Controller Plugin MUST implement this `GetVolume` RPC call if it has `GET_VOLUME` capability.
+
+A Controller Plugin MUST support the `volume_health` field in the `status` field in `GetVolumeResponse` if it has `GET_VOLUME_HEALTH` capability.
```

Comment:
I question the MUST wording here. The CO needs to be programmed defensively against UNKNOWN health conditions anyway, so why force a validation case here. IOW how will the CO reason differently about a missing health object (when VOLUME_HEALTH cap is reported) vs. a health object w/ UNKNOWN volume condition?

Instead, I'd like for us to consider defining `volume_health` as *always* optional, with the wording stating that plugins SHOULD provide a non-empty `volume_health` field if they advertise the `VOLUME_HEALTH` capability. This steers plugins toward the behavior that we want. The CO still has to handle both of the aforementioned cases, but neither case is now perceived an an "error" on the plugin's part.

Reply:
Sure